<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TournamentSettingsWidget</class>
 <widget class="QWidget" name="TournamentSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tournament type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="m_roundRobinRadio">
          <property name="toolTip">
           <string>Round-robin tournament</string>
          </property>
          <property name="text">
           <string>Ro&amp;und Robin</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">m_tournamentTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_gauntletRadio">
          <property name="toolTip">
           <string>First engine (or group of seeded engines) plays against the rest</string>
          </property>
          <property name="text">
           <string>Gaunt&amp;let</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">m_tournamentTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_knockoutRadio">
          <property name="toolTip">
           <string>Single-elimination tournament</string>
          </property>
          <property name="text">
           <string>Kno&amp;ckout</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">m_tournamentTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_pyramidRadio">
          <property name="toolTip">
           <string>Every engine plays against all of its predecessors</string>
          </property>
          <property name="text">
           <string>&amp;Pyramid</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">m_tournamentTypeGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_seedsSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>At most this many players from the top of the players list will be seeded in the tournament.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Seeds:</string>
          </property>
          <property name="buddy">
           <cstring>m_seedsSpin</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Rounds</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Rounds:</string>
        </property>
        <property name="buddy">
         <cstring>m_roundsSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="m_roundsSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Games per encounter:</string>
        </property>
        <property name="buddy">
         <cstring>m_gamesPerEncounterSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="m_gamesPerEncounterSpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Play each opening:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="GameRepetitionSpinBox" name="m_repeatSpin">
        <property name="toolTip">
         <string>Depends on &quot;games per encounter&quot;</string>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="specialValueText">
         <string>once</string>
        </property>
        <property name="suffix">
         <string> times</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Wait between games:</string>
        </property>
        <property name="buddy">
         <cstring>m_waitSpin</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="m_waitSpin">
        <property name="suffix">
         <string> s</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="m_recoverCheck">
        <property name="toolTip">
         <string>If enabled, crashed engines are restarted and the tournament is not stopped.</string>
        </property>
        <property name="text">
         <string>Recover crashed engines</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="m_saveUnfinishedGamesCheck">
        <property name="text">
         <string>Save unfinished games</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="m_swapCheck">
        <property name="toolTip">
         <string>Swap colors after each game within an encounter (default: true)</string>
        </property>
        <property name="text">
         <string>Swap sides</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Results</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>29</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="m_resultFormatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_resultFormatCombo"/>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="m_resultFormatEdit">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>70</y>
        <width>400</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="maxLength">
       <number>256</number>
      </property>
      <property name="placeholderText">
       <string>Result Format</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameRepetitionSpinBox</class>
   <extends>QSpinBox</extends>
   <header>src/gamerepetitionspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="m_tournamentTypeGroup"/>
 </buttongroups>
</ui>
